Stratis.Bitcoin.Tests/RPC/Models/TransactionModelsTest.cs

```diff
@@ -1,51 +1,48 @@
+            {
+                return new TransactionModelsTestCreateTransactions();
+            }
+
+            public TransactionModelsTestCreateTransactions()
+            {
+                this.txBlock10CoinbaseModelBrief = new TransactionBriefModel(Transaction.Parse(txBlock10Hex));
+                this.txBlock460373CoinbaseModelVerbose = new TransactionVerboseModel(Transaction.Parse(txBlock460373CoinbaseHex), Network.Main);
+                this.txTwoInTwoOutModelVerbose = new TransactionVerboseModel(Transaction.Parse(txTwoInTwoOutHex), Network.Main);
+            }
+
+
+            public const string txBlock10Hex = "01000000010000000000000000000000000000000000000000000000000000000000000000ffffffff0704ffff001d0136ffffffff0100f2052a01000000434104fcc2888ca91cf0103d8c5797c256bf976e81f280205d002d85b9b622ed1a6f820866c7b5fe12285cfa78c035355d752fc94a398b67597dc4fbb5b386816425ddac00000000";
+            public const string txBlock10Hash = "d3ad39fa52a89997ac7381c95eeffeaf40b66af7a57e9eba144be0a175a12b11";
+            public TransactionBriefModel txBlock10CoinbaseModelBrief;
+
+            public const string txBlock460373CoinbaseHex = "01000000010000000000000000000000000000000000000000000000000000000000000000ffffffff1e0355060704eba7e3582f4254432e434f4d2fb6000ddbcbe5000000000000ffffffff02a0ede0500000000017a914e083685a1097ce1ea9e91987ab9e94eae33d8a13870000000000000000266a24aa21a9ede6c8871b5b06ac4f8735c0bff5ebe3ca53d030e153a2735710535c78f2e71fdc00000000";
+            public const string txBlock460373CoinbaseHash = "ed6cadd7028e1406b766a57b14998f47ce5c44ec594ce7c26a6c5eab98f2e8dd";
+            public TransactionVerboseModel txBlock460373CoinbaseModelVerbose;
+
+            public const string txTwoInTwoOutHex = "0100000002b1a7582dad5e18960e7a8c0f3d9193fbb30f0c47b8114202eee31074f23bd0190e0000006b483045022100a050938989d6204761e473dedcaf61c029aca04a8c6d8d8c2d7c4357b23bc877022011ef04f87f9d90ce44421f44d0f4307949780ad5706701bdd8e350a5c31a3138012102077645ad77f8231f022d8801e956897f0e6a241f9be6b1ffde9ff5c7953d7af9ffffffffa9165ba655be36d78f4cd9522189cdf5dc4fb9fb796bb058c200c403930da2da000000006b483045022100919c8102e45f0ae9095b8052aa1888d616d72f2ec40cf54defd84d9fab22bda802207a724bacd9e9a0cf47a1f74e294dd597d849cc2a129b8add98957d37a721cddf012103c161ca4ac16fba4cca3f506acafb97e01cb55e95ea672054f2af566e2a35c8b0ffffffff02c4311723000000001976a91469d0a322604dcb268163764b9b476e33ee366cbc88ac28896310000000001976a914034978ca44c994ee7a80e1de11f2371809fac48f88ac00000000";
+            public const string txTwoInTwoOutHash = "951f379563ab5631dd2d249f06e76fe0ad19c03caabb22db2352f47ddc51fc31";
+            public TransactionVerboseModel txTwoInTwoOutModelVerbose;
         }
 
         [Fact]
         public void TransactionModelBriefRenderTest()
         {
-            var model = this.txBlock10CoinbaseModelBrief;
+            var model = TransactionModelsTestCreateTransactions.Create().txBlock10CoinbaseModelBrief;
```

Comment:
the create method seems to construct all 3 transaction models.
Could we have CreateTxBlock10CoinbaseModelBrief, CreateTxBlock460373CoinbaseModelVerbose instead? Just to avoid calling the constructor and initializing all the objects for every test.